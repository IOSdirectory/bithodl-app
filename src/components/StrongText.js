import React, { Component } from 'react';
import { StyleSheet, Text } from 'react-native';
import PropTypes from 'prop-types';

const styles = StyleSheet.create({
  strong: {
    fontWeight: '600'
  }
});

export default class StrongText extends Component {
  render() {
    return (
      <Text style={styles.strong}>
        {this.props.children}
      </Text>
    );
  }
}

StrongText.propTypes = {
  children: PropTypes.node
};
