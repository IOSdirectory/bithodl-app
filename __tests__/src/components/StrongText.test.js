import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';
import StrongText from '../../../src/components/StrongText';

describe('StrongText', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <StrongText>
        <Text>27125870-9efa-4ddd-b8fb-4f520751cd1d</Text>
      </StrongText>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
