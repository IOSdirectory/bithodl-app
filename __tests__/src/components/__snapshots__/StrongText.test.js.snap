// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StrongText renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  disabled={false}
  ellipsizeMode="tail"
  style={
    Object {
      "fontWeight": "600",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    27125870-9efa-4ddd-b8fb-4f520751cd1d
  </Text>
</Text>
`;
